source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "Jak psát zajímavé inzeráty pro ajťáky"
+perex: '''
+    Už pár let sleduji pracovní PHP trh z pozice programátorů i z pohledu firem. S oběma stranami mám dobrý vztah. Tuším, co potřebují a co je trápí. <strong>Vidím, že se navzájem chtějí, jen komuniace trochu vázne</strong>.<br><br>Původně z toho měl být feedback na inzeráty Jobs.cz a Skrz.cz. Při jejich čtení mě ale napadlo <strong>spoustu dalších tipů, o které se chci podělit</strong>. Dnes se mrkneme na jejich inzeráty a jak je dělat líp.
+'''
+reviewed_by: [3, 4]
+---
+
+
+## Čím začneme?
```

Comment:
Upozornil bych i na neznalost/nepřesnost. Je málo věcí, co může shodit inzerát tak, jako je neznalost. Jasně, HR není technický člověk, ale tak ať tu technickou část napíše IT guy. Velmi často vidím flame pod inzeráty typu "Hledáme programátora HTML a PHP".